### PR TITLE
docs(core): adds detailed information to directive metadata

### PR DESCRIPTION
**Pull request by ErickXavier**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?
Adds more detailed information about the templateUrl and styleUrls metadatas.

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
Doest talk anything about the field being possible to be relative paths and/or absolute urls.

Issue Number: N/A

## What is the new behavior?
Now it describes the fasct that the paths can be relative and the urls can be absolute.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by jenbuzz — CHANGES_REQUESTED** [review]

File: packages/core/src/metadata/directives.ts [hunk 1]
```diff
@@ -482,8 +482,8 @@ export interface Component extends Directive {
   moduleId?: string;
 
   /**
-   * The URL of a template file for an Angular component. If provided,
-   * do not supply an inline template using `template`.
+   * The relative path or absolute URL of a template file for an Angular component. If provided,
+   * do not supply ainline template using `template`.
```
[the remark this answers is not in the document]
Comment (ErickXavier) [reply packages/core/src/metadata/directives.ts]: Fixed. Thanks for the warning.

File: packages/core/src/metadata/directives.ts [hunk 2]
```diff
@@ -482,8 +482,8 @@ export interface Component extends Directive {
   moduleId?: string;
 
   /**
-   * The URL of a template file for an Angular component. If provided,
-   * do not supply an inline template using `template`.
+   * The relative path or absolute URL of a template file for an Angular component.
+   * If provided, do not supply ai nline template using `template`.
```
Comment (mhevery) [inline packages/core/src/metadata/directives.ts]: `ai nline` is a typo, maybe `an inline`?

Comment (jbogarthyde) [reply packages/core/src/metadata/directives.ts]: "an inline"

**Review by jbogarthyde — CHANGES_REQUESTED** [review]
Please correct typo and non-compliant commit message. 
Otherwise fine.

**Review by jbogarthyde — APPROVED** [review]

**Review by jenbuzz — APPROVED** [review]

**Review by brandonroberts — APPROVED** [review]

**Comment by brandonroberts** [comment]
Caretaker note: This is a docs only change and should not impact g3. Please run presubmit.

**Pull request closed without merging**